dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -582,7 +583,7 @@ private boolean isCorrectToolClass(String toolClass, String daoToolClass) {
             if (description != null && (tool.getDescription() == null || !tool.getDescription().contains(description))) {
                 return null;
             }
-            if (author != null && (tool.getAuthor() == null || !tool.getAuthor().contains(author))) {
+            if (author != null && (tool.getAuthors().isEmpty() || tool.getAuthors().stream().map(Author::getName).noneMatch(author::equals))) {
```

Comment:
Before it was doing a `contains`, now it's an `equals`. Is that intentional? Before, if the author was "Charles Overbeck", it looks like searching for "Charles" would find it, now it wouldn't?

Reply:
Good catch, I changed it back to `contains`